### ISSUE TITLE
Implement Synthetic Italic Support

### ISSUE THREAD

**Issue by Gankro** [servo/webrender#1654]
Some fonts don't natively support italics, and force the browser to synthesize them. The synthesis is fairly simple: apply a skew transform to the glyphs. 

"synthetic italics" should be made a font instance key, as introduced in https://github.com/servo/webrender/pull/1602

I believe the following requires synthetic italics on macos (not sure about other platforms):

```html
<span style='font-family: "Lucida Grande"'>
ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
</span>
```

**Comment by glennw** [comment]
@lsalzman Is this handled on all platforms now?

**Comment by lsalzman** [comment]
This will be fixed by PR #2245

**Comment by glennw** [comment]
#2245 has merged, closing as per comment above.
